release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -148,6 +148,90 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-vsphere-encryption-support"]
+==== VMware vSphere encryption support is generally available
+You can encrypt virtual machines (VMs) and persistent volumes (PVs) on {product-title} running on vSphere. This feature allows user-provisioned persistent storage to be encrypted over wire.
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-vpshere-top-support-multi-datacenters"]
+==== VMware vSphere CSI topology support for multiple datacenters is generally available
+{product-title} 4.12 introduced the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple compute clusters, thus helping to avoid a single point of failure. {product-title} 4.13 introduces support for deploying over multiple datacenters also.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-top-aware_persistent-storage-csi-vsphere[vSphere CSI topology].
+
+[id="ocp-4-13-storage-multiple-default-sc"]
+==== Creating more than one default storage class is generally available
+{product-title} 4.13 allows you create more than one default storage class. This feature makes it easier to change the default storage class because you can create a second storage class defined as the default. You then temporarily have two default storage classes before removing default status from the previous default storage class. While it is acceptable to have multiple default storage classes for a short time, you should ensure that eventually only one default storage class exists.
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-managing-default-sc"]
+==== Managing the default storage class is generally available 
+{product-title} 4.13 introduces the `spec.storageClassState` field in the `ClusterCSIDriver` object, which allows you to manage the default storage class to accomplish several different objectives:
+
+* Enforcing static provisioning by disabling dynamic provisioning.
+
+* When you have other preferred storage classes, preventing the storage operator from re-creating the initial default storage class.
+
+* Renaming, or otherwise changing, the default storage class
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-retroactive-default-sc-assignment"]
+==== Retroactive default StorageClass assignment (Technology Preview)
+There are some cases where the default storage classes are not assigned, or more than one default is defined. This feature allows you to retroactively assign a default storage class to correct these situations. This feature is supported with Technology Preview status.
+
```

Comment:
https://issues.redhat.com/browse/STOR-1061
This story works when no default storage class exists.

Reply:
@lpettyjo: As @chao007 pointed out: this feature is for cases when no default was set, more specifically if it was not set when PVC got created. This results in a PVC having no class which will get stuck in Pending phase. And with this feature enabled those PVC will get updated with default storage class **when/if** a default gets set later. Just a side node here: older versions than 4.13 did leave the PVC without storage class also if there was more than one default set, now one of them will be picked and assigned to PVC upon creation, the pick appears to be somewhat random to the user (it is age based, I would not dive into that here). But I think we could be more talkative here and explain both behavior changes briefly. I would write something like:

_There are some cases where it is desired to have more default storage classes configured or have no default set, for example when you need to transition to a different default storage class in your cluster. Creating any PVC (with no explicit storage class) while one of these configurations are in effect will cause that those PVCs will have no storage class assigned and dynamic provisioning will not take place without further manual intervention. OpenShift 4.13 improves this behavior so that having multiple default storage classes set will cause one class to be picked during PVC creation. This Technology Preview feature focuses on the second case and allows you to safely have no default set letting new PVCs be created without any assigned default. Those PVCs will remaining in Pending state and later, if you set a default storage class, those PVCs will get updated automatically with the current default and dynamic provisioning will take place._

Reply:
The feature description is indeed confusing. It does not cover the case where "more than one default is defined".
Also it doesn't really "allow you" since it is automatic.